Modules/OrgUnit/interfaces/class.ilOrgUnitPositionAccessHandler.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+/**
+ * Class ilOrgUnitPositionAccessHandler
+ *
+ * @author Fabian Schmid <fs@studer-raimann.ch>
+ */
+interface ilOrgUnitPositionAccessHandler {
+
+	const PERMISSION_VIEW_LEARNING_PROGRESS = 'viewlp';
+	const PERMISSION_VIEW_TEST_RESULTS = 'viewtstr';
```

Comment:
Absolutely, this should just help to name "things", but a more general way should be gone in a next release